Jenkinsfile.yml

```diff
@@ -16,12 +17,14 @@ node('ubuntu18.04-OnDemand'){
       sh 'git clone "${REPO_PRI}"'
       sh 'git clone "${REPO_SCT}"'
```

Comment:
Missing Rollover in container/tomcat/conf/catalina.properties. Line 133

Reply:
Done.